Jellyfin.Xtream/SeriesChannel.cs

```diff
@@ -131,7 +131,7 @@ private ChannelItemInfo CreateChannelItemInfo(Series series)
         return new ChannelItemInfo()
         {
             CommunityRating = (float)series.Rating5Based,
-            DateModified = series.LastModified,
+            DateModified = series.LastModified ?? DateTime.MinValue,
```

Comment:
This most likely will cause issues with updated data. A better default would be the current date and force an update on every refresh.